docs/scripts/buildApi.ts

```diff
@@ -857,7 +857,7 @@ async function buildDocs(options: {
         description += ' See <a href="#css">CSS API</a> below for more details.';
       } else if (propName === 'sx') {
         description +=
-          ' See the <a href="/system/basics/#the-sx-prop">`sx` page</a> for more details.';
+          ' See the <a href="/system/the-sx-prop">`sx` page</a> for more details.';
```

Comment:
The leading dash avoid a 301

```suggestion
          ' See the <a href="/system/the-sx-prop/">`sx` page</a> for more details.';
```

```sh
$ curl -I https://next.material-ui.com/system/basics\#the-sx-prop
HTTP/2 301
cache-control: public, max-age=0, must-revalidate
```

x the other cases